### ISSUE TITLE
Recent notebook list doesn't save frequently enough

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#393]
```
Preferences are not saved when recent notebook list is changed.  It can cause 
the list to not be saved when the program exits.  Perhaps save whenever the 
list changes or force a save when the program exits.

```

Original issue reported on code.google.com by `matt.rasmus@gmail.com` on 7 Jan 2011 at 5:38

**Comment by GoogleCodeExporter** [comment]
```
Should be solved by keepnote-0.6.8

http://keepnote.org/keepnote/download-test/

```

Original comment by `matt.rasmus@gmail.com` on 23 Jan 2011 at 3:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 23 Feb 2011 at 4:05
- Changed state: **Done**
